src/mono/mono/mini/llvm-intrinsics.h

```diff
@@ -283,6 +283,18 @@ INTRINS_OVR(AARCH64_ADV_SIMD_ABS_INT8, aarch64_neon_abs, sse_i1_t)
 INTRINS_OVR(AARCH64_ADV_SIMD_ABS_INT16, aarch64_neon_abs, sse_i2_t)
 INTRINS_OVR(AARCH64_ADV_SIMD_ABS_INT32, aarch64_neon_abs, sse_i4_t)
 INTRINS_OVR(AARCH64_ADV_SIMD_ABS_INT64, aarch64_neon_abs, sse_i8_t)
+INTRINS_OVR(AARCH64_ADV_SIMD_ABS_SATURATE_INT8, aarch64_neon_sqabs, sse_i1_t)
+INTRINS_OVR(AARCH64_ADV_SIMD_ABS_SATURATE_INT16, aarch64_neon_sqabs, sse_i2_t)
+INTRINS_OVR(AARCH64_ADV_SIMD_ABS_SATURATE_INT32, aarch64_neon_sqabs, sse_i4_t)
+INTRINS_OVR(AARCH64_ADV_SIMD_ABS_SATURATE_INT64, aarch64_neon_sqabs, sse_i8_t)
```

Comment:
Why the int64 overload here? It looks like we only have SByte, Int16, and Int32 in the public API.

Reply:
Its a slight quirk, but we are actually handling both the AdvSimd and AdvSimd.Arm64 classes here. The AdvSimd.Arm64 has the version for Int64:

 https://docs.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.arm.advsimd.arm64.abssaturate?view=net-5.0#System_Runtime_Intrinsics_Arm_AdvSimd_Arm64_AbsSaturate_System_Runtime_Intrinsics_Vector128_System_Int64__

Reply:
Gotcha, thanks.